packages/common/src/ui/components/menus/ColorMenu/ColorMenu.tsx

```diff
@@ -0,0 +1,75 @@
+import { Paper, Popover } from '@material-ui/core';
+import React, { FC } from 'react';
+import { Circle } from '../../../icons';
+
+export interface Color {
+  hex: string;
+  name: string;
+}
+
+interface ColorMenuProps {
+  colors?: Color[];
+  anchorEl: HTMLButtonElement | null;
+  onClick: (color: Color) => void;
+  onClose: () => void;
+}
+
+const defaultColors = [
+  { hex: '#004fc4', name: 'blue' },
+  { hex: '#05a660', name: 'green' },
+  { hex: '#ff3b3b', name: 'red' },
+  { hex: '#fc0', name: 'yellow' },
+  { hex: '#00b7c4', name: 'aqua' },
+  { hex: '#8f90a6', name: 'grey' },
+];
```

Comment:
Making this have a max width for when you have many options, with some line wrapping would have been good